testing/tools/integration/integration.py

```diff
@@ -756,10 +756,34 @@ def ex_validate(cmd):
 def setup_resilience_path(res_dir):
     # create resilience data directory if it doesn't already exist
     if not os.path.exists(res_dir):
+        create_resilience_dir(res_dir)
+    empty_resilience_dir(res_dir)
+
+def create_resilience_dir(res_dir):
+    try:
         os.mkdir(res_dir)
+    except Exception as e:
+        print 'Warning: mkdir %s failed: %s' % (res_dir, e)
```

Comment:
You should replace the print with `logging.exception('Warning: mkdir %s failed' % res_dir)`, which will reraise the error properly (causing a test failure, as it should).